plugins/repository-azure/src/main/java/org/elasticsearch/cloud/azure/storage/AzureStorageSettings.java

```diff
@@ -43,20 +45,43 @@
         Setting.affixKeySetting(Storage.PREFIX, "key", (key) -> Setting.simpleString(key, Setting.Property.NodeScope));
     private static final Setting<Boolean> DEFAULT_SETTING =
         Setting.affixKeySetting(Storage.PREFIX, "default", (key) -> Setting.boolSetting(key, false, Setting.Property.NodeScope));
+    /** The host name of a proxy to connect to s3 through. */
```

Comment:
Nit: copy & paste error: "The host name of a proxy to connect to **azure** through." (also: shouldn't it be "Azure" instead of "azure"?)